Datastores/functions/pto_demo.ts

```diff
@@ -0,0 +1,178 @@
+import { DefineFunction, Schema, SlackFunction } from "deno-slack-sdk/mod.ts";
+import PTO from "../datastores/pto.ts";
+
+export const def = DefineFunction({
+  callback_id: "datastore-ttl-demo",
+  title: "Datastore TTL demo",
+  source_file: "Datastores/functions/pto_demo.ts",
+  input_parameters: {
+    properties: {
+      interactivity: { type: Schema.slack.types.interactivity },
+      user_id: { type: Schema.slack.types.user_id },
+    },
+    required: ["interactivity", "user_id"],
+  },
+  output_parameters: { properties: {}, required: [] },
+});
+
+export default SlackFunction(def, async ({ inputs, client }) => {
+  const offset = (await client.users.info({
+    user: inputs.user_id,
+  })).user?.ts_offset || 0;
+
+  const {
+    items: allItems,
+    response_metadata,
+  } = await client.apps.datastore.query({ datastore: PTO.name });
+  let cursor = response_metadata?.next_cursor;
+  while (cursor) {
+    const {
+      paginationItems,
+      response_metadata,
+    } = await client.apps.datastore.query({ datastore: PTO.name });
+    allItems.push(paginationItems);
```

Comment:
```suggestion
      items,
      response_metadata,
    } = await client.apps.datastore.query({ datastore: PTO.name, cursor });
    allItems.push(...items);
```

Small change since I think `items` is always returned here, but might also be empty and push a `[]`. And also a `cursor` to avoid loops!